project-repo/templates/common/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Report an issue
+description: Create a bug report to fix an issue
+title: ""
+labels: ""
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to report an issue and help improve PROJECT_TODO! Please fill out the form below as
+        best as you can so that we can help you.
+
+        Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general
+        support or usage questions, use [Ubuntu Discourse](https://discourse.ubuntu.com/c/desktop/8).
+
+        By submitting an Issue to this repository, you agree to the terms within the
+        [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct).
+
+        Be careful with sensitive information and security vulnerabilities. In order to report bugs that could contain
+        sensitive information, use [Launchpad](https://bugs.launchpad.net/ubuntu/+source/PROJECT_TODO/+filebug) instead.
+        On Ubuntu machines, you can use `ubuntu-bug PROJECT_TODO` to collect relevant information.
```

Comment:
I think the previous example was a little bit more clear in that regard. This needs to be at the top to warn the user. Also, it had a disctinct line to say it’s important:
```
>**Please do not report security vulnerabilities here**
>
> Please use [launchpad private bugs](https://bugs.launchpad.net/ubuntu/+source/PROJECT_TODO/+filebug) which is monitored by our security team. On ubuntu machine, it’s best to use `ubuntu-bug PROJECT_TODO` to collect relevant information.
> FIXME: snap?
```

So, with your new better wording, to you mind moving that back to the top?

Reply:
There's also an argument to be made that we should thank the user for sparing the time to report the issue, no?

I agree on the marking, though. It's better to be more explicit.

Reply:
Right, but the main argument is "the most important information on top". I think it’s fine to tell the user "stop here, please read this, ok to go on, then Thanks for reporting this bug and blablabla".

This is why the previous template had that, and it’s followed up in many ticketting system (launchpad, track, gitlab…)